client/components/upstreamMessages/html/src/minimal/app.js

```diff
@@ -0,0 +1,31 @@
+async function onPayPalLoaded() {
+  try {
+    const clientToken = await getBrowserSafeClientToken();
+    const sdkInstance = await window.paypal.createInstance({
+      clientToken,
+      components: ["paypal-messages"],
+    });
+    createMessage(sdkInstance);
+  } catch (error) {
+    console.error(error);
+  }
+}
+
+async function getBrowserSafeClientToken() {
+  const response = await fetch("/paypal-api/auth/browser-safe-client-token", {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+  const { accessToken } = await response.json();
+
+  return accessToken;
+}
+
+async function createMessage(sdkInstance) {
+  sdkInstance.createPayPalMessages({
+    buyerCountry: "US",
+    currencyCode: "USD",
+  });
```

Comment:
Since this is minimal, maybe we can include the `currency-code` as an attribute on the web component and then leave these options empty:

```suggestion
  sdkInstance.createPayPalMessages();
```